Source/Core/DolphinWX/Debugger/CodeView.cpp

```diff
@@ -372,6 +393,7 @@ void CCodeView::OnMouseUpR(wxMouseEvent& event)
   menu.Append(IDM_JITRESULTS, _("PPC vs x86"))->Enable(Core::IsRunning());
   menu.Append(IDM_INSERTBLR, _("Insert &blr"))->Enable(Core::IsRunning());
   menu.Append(IDM_INSERTNOP, _("Insert &nop"))->Enable(Core::IsRunning());
+  menu.Append(IDM_ASSEMBLE, _("Asse&mble"))->Enable(Core::IsRunning());
```

Comment:
Should this be more indicative of what it does (i.e. "Replace Opcode"/"Replace Instruction"/w.e. is appropriate) or something? It seems odd to use "Assemble" in the context of something that's already 'assembled'.